steps/step-2/src/main/java/vertx/APIClientVerticle.java

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
I don't think we should reformat this file as it will affect the rendering of the how to.

Reply:
Okay, sorry... my IDE did that automagically. I'll roll back the formatting stuff, but keep the small code change(?)